layouts/partials/page-edit.html

```diff
@@ -0,0 +1,38 @@
+{{ $ctx := .ctx }}
+{{ $type := .type}}
+{{ $.Scratch.Set "editURL" "" }}
+
+{{ if $ctx.Params.dependencies }}
+{{/* Files with dependencies link to corresponding URL */}}
+
+    {{ $deps := $ctx.Params.dependencies }}
+    {{ $depsIsArray := reflect.IsSlice $deps }}
+
+    {{ if $depsIsArray }}
+
+        {{ range $dep := $deps }}
+
+                {{ with $ctx.File }}
+                    {{ partial "page-edit-body.html" (dict "ctx" $ctx "link" $dep "type" $type) }}
+                {{ end }}
+                
+        {{ end }}
+
+    {{ end }}
+
+{{ else if (eq $ctx.Section "integrations" )}}
+{{/* Files with integrations path and no dependencies link to new issue */}}
+
+    {{ $ctx.Scratch.Set "editURL" "https://github.com/DataDog/documentation/issues/new?template=issue.md" }}
```

Comment:
This needs to be merged after: https://github.com/DataDog/documentation/pull/6471

```suggestion
    {{ $ctx.Scratch.Set "editURL" "https://github.com/DataDog/documentation/issues/new?template=edit_this_page.md" }}
```